tests/target/issue-4909.rs

```diff
@@ -0,0 +1,70 @@
+// rustfmt-normalize_comments: true
+
```

Comment:
There was a comment reported on the linked issue that the same symptoms were observed with `wrap_comments` as well. Would you mind also checking if the original behavior can be reproduced with `wrap_comments`, and if so, whether that's also resolved with your fix (and include a test with `wrap_comments` too :pray:)

Reply:
Yes, the issue was present with `wrap_comments` too. I added more tests now.

Reply:
Fantastic! One last thing :smile: would you mind restructuring the tests to have separate pairs with `wrap_comments` and `normalize_comments` set in isolation of each other?

Reply:
Ohh I didn't realize it was also happening with `wrap_comments` in isolation. I will add another test.